config/kafka-instance-types-configuration.yaml

```diff
@@ -48,3 +48,17 @@ supported_instance_types:
       quotaConsumed: 1
       quotaType: "rhosak"
       capacityConsumed: 1
+  - id: eval
+    display_name: Eval
+    sizes:
+    - id: x1
+      ingressThroughputPerSec: "30Mi"
+      egressThroughputPerSec: "30Mi"
+      totalMaxConnections: 3000
+      maxConnectionAttemptsPerSec: 100
+      maxDataRetentionSize: "1000Gi"
+      maxDataRetentionPeriod: "P14D"
+      maxPartitions: 1000
+      quotaConsumed: 1
+      quotaType: "rhosak"
```

Comment:
`quotaType` should be "RHOSAKTrial"

Reply:
we're also assigning `rhosak` to developer instances. It should be the same here.

Reply:
The contents of the "eval" instance type are a copy of the contents I found in the "developer" type with the exception of the displayName and the id.
The reason for that is that in the larger-kafka-instances branch there was "eval" and it was renamed to "developer". If that's not accurate let me know as I don't have the context about this

Reply:
@pb82 @miguelsorianod : it should be:
```yaml
quotaConsumed: 0
quotaType: "RHOSAKTrial"
```

for both `eval` and `developer`

Reply:
Ok, got you. `quotaType` should refer to the Quoty (RHOSAK and RHOSAKTrial) and not to the resource (rhosak).

Reply:
Updated. For details see the new comments in the issue